machine-learning/lambda-v2/README.md

```diff
@@ -2,14 +2,20 @@
 
```

Comment:
L1: Inference applications that run AWS Lambda functions in a Greengrass V2 component